addons/purchase/models/purchase.py

```diff
@@ -726,36 +797,36 @@ def _get_product_purchase_description(self, product_lang):
 
         return name
 
-    def _prepare_account_move_line(self, move):
+    def _prepare_account_move_line(self, move=False):
```

Comment:
I guess that if the invoice creation allow to do batch of PO. Then the magical onchange would be useless and the onchange on account.move could be remove as the move kwarg on this function.

cc @oma-odoo

Reply:
We still want the onchange for PO if we directly come from accounting.